config/routes.rb

```diff
@@ -18,10 +17,12 @@
   resources :volunteers do
     get :seeking_clients, on: :collection
     get :find_client, on: :member, to: 'assignments#find_client'
+    get :volunteer_hours, on: :member
     resources :journals
   end
   resources :volunteer_emails
   resources :profiles, except: [:destroy, :index]
+  resources :hours
```

Comment:
I think something like that makes more sense:

```ruby
resources :assignments do
  resources :hours
end
resources :volunteers do
  resources :hours
end
```

This brings an automatic scope to either Volunteers hours, or Assignments hours.